entrypoint/airgap/airgap_suite_test.go

```diff
@@ -0,0 +1,52 @@
+package airgap
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/rancher/distros-test-framework/config"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	flags   *customflag.FlagConfig
+	cluster *shared.Cluster
+)
+
+func TestMain(m *testing.M) {
+	flags = &customflag.ServiceFlag
+	flag.Var(&flags.Destroy, "destroy", "Destroy cluster after test")
+	flag.StringVar(&flags.AirgapFlag.RegistryUsername, "registryUsername", "testuser", "private registry username")
+	flag.StringVar(&flags.AirgapFlag.RegistryPassword, "registryPassword", "testpass765", "private registry password")
+	flag.StringVar(&flags.AirgapFlag.TarballType, "tarballType", "", "artifact tarball type")
+	flag.Parse()
+
+	_, err := config.AddEnv()
+	if err != nil {
+		shared.LogLevel("error", "error adding env vars: %w\n", err)
+		os.Exit(1)
+	}
+
```

Comment:
can we add a validation for the var that needs to be set? 
as like https://github.com/rancher/distros-test-framework/blob/5ca09fae164a7a63d6f0ae0a1d8abecfd160988a/entrypoint/deployrancher/rancher_suite_test.go#L45

Reply:
yeah sure

Reply:
Updated